apps/react/src/features/genres/components/GenresFilterForm/GenresFilterForm.tsx

```diff
@@ -0,0 +1,20 @@
+import { memo, FC } from 'react';
+import { TextField } from '@mui/material';
+
+import styles from './GenresFilterForm.module.css';
+
+/** Genres filter form. */
+const GenresFilterFormComponent: FC = () => (
+	<form className={styles.form}>
+		<TextField
+			className={styles.form__field}
+			label="Search"
+			variant="outlined"
+		/>
+	</form>
+);
+
+/**
+ * Memorized filter form.
+ */
```

Comment:
Please prefer one-line comments for methods without parameters

```suggestion

/** Memorized filter form. */
```